changelog.mustache

```diff
@@ -0,0 +1,23 @@
+# Git Changelog plugin changelog
```

Comment:
What is this file for? A kind of configuration?
This should be placed in another directory then.